File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT2_JAVA/src/Organisms/Animals/Sheep.java
package Organisms.Animals;

import Game.World;

import java.awt.*;
import java.io.Serializable;

public class Sheep extends Animal implements Serializable {
    public Sheep(World world, int x, int y) {
        super("S", "Sheep", Color.LIGHT_GRAY, 4, 4, 1, world,x,y);
    }
    @Override
    public Sheep getChild(int x, int y)
    {
        return new Sheep(world,x,y);
    }
}

File: 2 SEMESTER/Object Oriented Programming/Projects/PROJECT3_PYTHON/Organisms/Animals/sheep.py
from Organisms.Animals.animal import Animal


class Sheep(Animal):
    def __init__(self, position_x, position_y, world):
        super().__init__(position_x, position_y, "Sheep", 'S', 4, 4, world, 1, (211, 211, 211))

    def getchild(self, newposition):
        return Sheep(newposition[0], newposition[1], self.world)